command/format_test.go

```diff
@@ -174,6 +175,7 @@ func getMockStatusData(emptyFields bool) SealStatusOutput {
 			ClusterID:    "cluster id",
 			RecoverySeal: true,
 			StorageType:  "storage type",
+			Warnings:     append(make([]string, 0), "warning"),
```

Comment:
How is this different from `[]string{"warning"}`?

Reply:
It's not, I just had a brain fart - fixing this!